ai-platform/snippets/quickstart.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+/**
+ * TODO: add an actual quickstart example.
+ */
+async function main() {
+  // [START aiplatform_quickstart_sample]
```

Comment:
Is this used at all? I tried searching for this region tag, but couldn't find anything relevant. 😞

Reply:
@ace-n For this particular sample set, we're migrating all even if they are not used yet.